dataprofiler/tests/profilers/profiler_options/test_profiler_class_options.py

```diff
@@ -0,0 +1,116 @@
+            self.assertIn(key, options.properties)
+
+    def test_set_helper(self):
+        super().test_set_helper()
+        option = self.get_options()
+        optpth = self.get_options_path()
+        
+        # Enable and Disable Option
+        for key in option.structured_options.properties:
+            option._set_helper({'structured_options.{}.is_enabled'.format(key): False}, '')
+            self.assertFalse(option.structured_options.properties[key].is_enabled)        
+            option._set_helper({'structured_options.{}.is_enabled'.format(key): True}, '')
+            self.assertTrue(option.structured_options.properties[key].is_enabled)        
+
+        # Treat is_enabled as a BooleanOption
+        for key in option.structured_options.properties:
+            expected_error = "type object 'structured_options.{}.is_enabled' has no attribute 'is_enabled'".format(key)
+            with self.assertRaisesRegex(AttributeError, expected_error):
+                option._set_helper({'structured_options.{}.is_enabled.is_enabled'.format(key): True}, '')    
+        
+    def test_set(self):
+        super().test_set()
+        option = self.get_options()
+        optpth = self.get_options_path()
+
+        # Enable and Disable Options        
+        for key in option.structured_options.properties:
+            option.set({'structured_options.{}.is_enabled'.format(key): False})
+            self.assertFalse(option.structured_options.properties[key].is_enabled)        
+            option.set({'structured_options.{}.is_enabled'.format(key): True})
+            self.assertTrue(option.structured_options.properties[key].is_enabled)        
+    
+        # Treat is_enabled as a BooleanOption
+        for key in option.structured_options.properties:
+            expected_error = "type object 'structured_options.{}.is_enabled' has no attribute 'is_enabled'".format(key)
+            with self.assertRaisesRegex(AttributeError, expected_error):
+                option.set({'structured_options.{}.is_enabled.is_enabled'.format(key): True})
+    
+    def test_validate_helper(self):
+        # Valid cases should return [] while invalid cases should return a list of errors
+        option = self.get_options()
+        optpth = self.get_options_path()
+
+        # Default Configuration Is Valid
+        self.assertEqual([], option._validate_helper())
+        
+        # Variable Path Is Not A String
+        expected_error = "The variable path must be a string."
+        with self.assertRaisesRegex(ValueError, expected_error):
+            option._validate_helper(1)
+        
+        # Option is_enabled is not a boolean
+        for key in option.structured_options.properties:
+            option.set({'structured_options.{}.is_enabled'.format(key): "Hello World"}) 
+
+        expected_error = ['{}.structured_options.{}.is_enabled must be a Boolean.'.format(optpth, key) 
+            for key in option.structured_options.properties]
```

Comment:
Indent this `for` to line up with the open bracket on the line above it

Reply:
Got it, will do